synfig-studio/src/gui/app.cpp

```diff
@@ -1347,8 +1346,8 @@ App::get_default_accel_map()
 
 	return default_accel_map;
 }
-Glib::RefPtr<App> App::create() {
-	app_reference = Glib::RefPtr<App>(new App());
+Glib::RefPtr<App> App::instance() {
+	Glib::RefPtr<studio::App> app_reference = Glib::RefPtr<App>(new App());
```

Comment:
In this case - yes, you will get a new object every time you call the `instance()` method. 
Correct code should look like:
```
static Glib::RefPtr<studio::App> app_reference = Glib::RefPtr<App>(new App());
```

Reply:
Changes will me made in the next commit :)